spec/controllers/admin/orders_controller_spec.rb

```diff
@@ -0,0 +1,70 @@
+require "rails_helper"
+
+# Test Authorization by using the Pundit concern and an example policy,
+# which will test all the authorization functionality.
+
+describe Admin::OrdersController, type: :controller do
+  context "with Punditize concern" do
+    
+    controller(Admin::OrdersController) do
+      include Administrate::Punditize
+      def pundit_user
+        Customer.first # assume the user is the first Customer
+      end
+    end
+    
+    let!(:user) { create :customer }
+    
+    before(:each) do
+      # Create a few orders for the user and a few for other customers
+      create_list :order, 4, customer: create(:customer)
+      create_list :order, 7, customer: user
+      create_list :order, 2, customer: create(:customer)
+      create_list :order, 2, customer: user
+    end
+    
+    # Policies are defined in order_policy.rb
+    describe 'GET index' do
+      it 'shows only the records in the admin scope' do
+        locals = capture_view_locals { get :index }
+        expect(locals[:resources].count).to eq(9) # only my orders
+      end
+    end
```

Comment:
Please separate these with blank lines. I'm surprised that Hound is not complaining about this and the single quotes?